src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertFrom-StringData.cs

```diff
@@ -54,10 +69,10 @@ protected override void ProcessRecord()
             {
                 string s = line.Trim();
 
-                if (String.IsNullOrEmpty(s) || s[0] == '#')
+                if (string.IsNullOrEmpty(s) || s[0] == '#')
                     continue;
 
-                int index = s.IndexOf('=');
+                int index = s.IndexOf(Delimiter);
```

Comment:
It was a char. Now it is string. I believe it should be char like csv cmdlets have.

Reply:
Addressed in most recent commit